app/ci_program.py

```diff
@@ -105,7 +105,7 @@ def commandLoop(self):
       while not len(cmdList):
         for i in range(5):
           ch = window.cursorWindow.getch()
-          if ch == curses.ascii.ESC:
+          if ch == app.curses_util.KEY_ESCAPE:
```

Comment:
I think this one should stay as curses.ascii.ESC because it's testing for the start of an escape sequence which has a particular value.